snap/info.go

```diff
@@ -1594,7 +1594,12 @@ var NewContainerFromDir func(snapName string) Container = func(snapName string)
 // ReadCurrentComponentInfo reads the ComponentInfo for the currently linked
 // revision of the given component associated with the given snap.
 func ReadCurrentComponentInfo(component string, info *Info) (*ComponentInfo, error) {
-	link := filepath.Join(ComponentsBaseDir(info.InstanceName()), info.Revision.String(), component)
+	// NOTE: creating this here is a bit of a hack, since we aren't actually
```

Comment:
s/NOTE/TODO, when we have a bit more time or other use cases we should reconsider this